### DISCUSSION TITLE
Is posible that Full finetune Llama3-8B on 2xA100 (40GB Vram per GPU)?

### DISCUSSION THREAD

**Discussion by anhnh2002** [pytorch/torchtune#1166] [Q&A] [answered]
Is posible that Full finetune Llama3-8B on 2xA100 (40GB Vram per GPU)? If so, how can i do it?

**Comment by pbontrager** [comment, marked as answer]
I believe that you could even finetune it on only one A100 for the 8B model. But for two gpus, use the 8B_full config which is for distributed recipes. You can launch with `tune run --nnodes=1 --nproc-per-node=2 full_finetune_distributed --config llama3/8B_full`. You can also `tune cp llama3/8B_full <my_config>` to be able to modify the config for a batch size, learning rate, etc that it optimal for your compute setup.
